src/calculateRentalCost.js

```diff
@@ -28,7 +28,22 @@
  */
 
 function calculateRentalCost(days) {
-  // write code here
+  const rent = 40;
+  let result = 0;
+
+  if (days < 3) {
+    result = rent * days;
```

Comment:
```suggestion
    return rent * days;
```

and you cant remove your result variable